### PR TITLE
Adding profile display test (com_users).

### PR DESCRIPTION
**Pull request by charvimehradu**
Pull Request for Issue # 

Adding user profile display test.

### CODE REVIEW

File: tests/System/integration/site/components/com_users/Profile_display.cy.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      .then(() => {
+        cy.visit('index.php?option=com_users&view=profile');
+
+        cy.get('#username').type('test');
+        cy.get('#password').type('test');
+        cy.get('#remember').check();
+        cy.get('.controls > .btn').click();
+
+        cy.get('#system-message-container').should('contain.text', 'You have been logged in.');
+        cy.get('#users-profile-core').should('contain.text', 'Profile');
```
Comment (laoneo) [inline tests/System/integration/site/components/com_users/Profile_display.cy.js]: Can you add some more asserts if the username is "test", etc.?

File: tests/System/integration/site/components/com_users/Profile_display.cy.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        cy.get('#users-profile-core').should('contain.text', 'Name');
+        cy.get('#users-profile-core').should('contain.text', 'test user');
```
Comment (laoneo) [inline tests/System/integration/site/components/com_users/Profile_display.cy.js]: Please give it a different name than the default one, when you call db_createUser and then test of that different name here.

Comment (charvimehradu) [reply tests/System/integration/site/components/com_users/Profile_display.cy.js]: I have made the requested change. Could you please review it and confirm if I have done it correctly? Thank you.

File: tests/System/integration/site/components/com_users/Profile_display.cy.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        cy.get('.controls > .btn').click();
+
+        cy.get('#system-message-container').should('contain.text', 'You have been logged in.');
+        cy.get('#users-profile-core').should('contain.text', 'Profile');
+        cy.get('#users-profile-core').should('contain.text', 'Name');
+        cy.get('#users-profile-core').should('contain.text', 'automated test user');
+        cy.get('#users-profile-core').should('contain.text', 'Username');
+        cy.get('#users-profile-core').should('contain.text', 'test');
+        cy.get('#users-profile-core').should('contain.text', 'Registered Date');
+        cy.get('#users-profile-core').should('contain.text', 'Wednesday, 01 March 2023');
```
Comment (laoneo) [inline tests/System/integration/site/components/com_users/Profile_display.cy.js]: When you test this, then you need to pass the respective variable to db_createUser.

**Comment by laoneo** [comment]
Looks good, just a minor comment. can you do that then for the menu item test as well?

**Comment by laoneo** [comment]
Can you remove the _display part from the filename and do the check with a different username. How it is now, you are testing the name with the username test.

**Comment by charvimehradu** [comment]
Thank you for bringing this to my attention; I have now corrected it.

**Pull request merged**